project/project.go

```diff
@@ -136,8 +136,8 @@ func (p *Project) Load(bytes []byte) error {
 	return nil
 }
 
-func (p *Project) loadWrappers(wrappers map[string]*serviceWrapper) error {
-	for _, name := range p.reload {
+func (p *Project) loadWrappers(wrappers map[string]*serviceWrapper, list []string) error {
+	for _, name := range list {
```

Comment:
What's in this list? could we get a better variable name or a comment about the contents?